atlasdb-commons/src/main/java/com/palantir/common/base/Throwables.java

```diff
@@ -82,7 +82,8 @@ public static RuntimeException rewrapAndThrowUncheckedException(String newMessag
      * If Throwable is a RuntimeException or Error, rethrow it. If not, throw a
      * new PalantirRuntimeException(ex)
      */
-    public static RuntimeException throwUncheckedException(Throwable ex) {
+    public static RuntimeException
```

Comment:
This formatting change doesn't appear related to this PR.  Nor does it appear in line with our regular formatting.